x-pack/plugins/observability_solution/observability_ai_assistant/server/routes/connectors/route.ts

```diff
@@ -20,9 +20,22 @@ const listConnectorsRoute = createObservabilityAIAssistantServerRoute({
       await plugins.actions.start()
     ).getActionsClientWithRequest(request);
 
-    const connectors = await actionsClient.getAll();
+    const [availableTypes, connectors] = await Promise.all([
+      actionsClient
+        .listTypes({
+          includeSystemActionTypes: false,
+        })
+        .then((types) =>
+          types.filter((type) => type.enabled && type.enabledInLicense).map((type) => type.id)
```

Comment:
@cnasikas there is also `enabledInConfig`, but I wasn't sure what that means. Do we also need to take it into account?

Reply:
Good question. Users can disable connector types in `kibana.yml` (`xpack.actions.enabledActionTypes`). Better to filter them out.